dma/imx6ull-sdma/imx6ull-sdma.c

```diff
@@ -641,9 +643,15 @@ static int dev_read(oid_t *oid, void *data, size_t size)
 {
 	int channel = oid_to_channel(oid);
 	unsigned intr_cnt;
+	int res;
 
 	mutexLock(common.lock);
-	condWait(common.channel[channel].intr_cond, common.lock, 0);
+	res = condWait(common.channel[channel].intr_cond, common.lock, INTR_CHANNEL_TIMEOUT_US);
```

Comment:
Nitpick - this condWait is while-less, and signal can cause spurious wakeup. So maybe do ... while (ret == -EINTR)?